NuGet.Extensions/ReferenceAnalysers/ProjectNugetifier.cs

```diff
@@ -31,7 +31,7 @@ public ProjectNugetifier(IVsProject vsProject, IPackageRepository packageReposit
 
         public ICollection<IPackage> NugetifyReferences(DirectoryInfo solutionDir)
         {
-            var binaryReferences = _vsProject.GetBinaryReferences().ToList();
+            var binaryReferences = GetReferences().ToList();
```

Comment:
Yep it's used in the first commit of PR https://github.com/BenPhegan/NuGet.Extensions/pull/37